test/packages/other/system_test_assert_conditions/data_stream/test/_dev/test/system/test-fields_present-config.yml

```diff
@@ -0,0 +1,33 @@
+vars: ~
+data_stream:
+  vars:
+    paths:
+      - "/custom/paths/logs.json"
+assert:
+  fields_present:
+    - target.file
+    - target.expected
+    - target.finish
```

Comment:
Wdyt about adding a false positives test package to check that it fails if not satisfied?

Reply:
Added a test package (false positive) to test `assert.fields_present`
https://github.com/elastic/elastic-package/pull/2429/commits/492f6cacba390318301ef5dcf46702673dffcc29